etabotsite/etabotapp/models.py

```diff
@@ -70,4 +70,49 @@ def validate_tms_credential(sender, instance, **kwargs):
                         {})
     TMS_w1.connect_to_TMS(instance.password)
     logging.debug('validate_tms_credential finished')
+    logging.debug('instance  ({}): {}'.format(type(instance), instance))
```

Comment:
Dummy test code shouldn't present here. If you need dummy data for testing, please add it to unit test (this is why unit test exist :-) )

Reply:
Well, this is dummy test data for testing UI by a human, so unit test wouldn't work. We can use a separate branch for it though